components/shared/CollectionDeposit.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <i18n path="tooltip.deposit" tag="p">
+      <template #amount>
+        <strong>{{ requiredAmount }}</strong>
+      </template>
+      <template #token>
+        <strong>{{ token }}</strong>
+      </template>
+    </i18n>
+    <a v-safe-href="learnMoreLink" class="learn-more">{{
+      $t('helper.learnMore')
+    }}</a>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  COLLECTION_DEPOSIT_BSX,
+  COLLECTION_DEPOSIT_KSM,
+} from '~~/utils/constants'
+
+const { urlPrefix } = usePrefix()
+
+const token = urlPrefix.value.toUpperCase()
+const requiredAmount =
+  token === 'KSM' ? COLLECTION_DEPOSIT_KSM : COLLECTION_DEPOSIT_BSX
+const learnMoreLink =
+  token === 'KSM'
+    ? 'https://hello.kodadot.xyz/multi-chain/fees/assethub-fees#polkadot-asset-hub-fees-prev.-statemint'
+    : 'https://hello.kodadot.xyz/multi-chain/fees/assethub-fees#polkadot-asset-hub-fees-prev.-statemint?'
```

Comment:
?

Reply:
Based on previous comments I was under the impression that there should be a different link based on `urlPrefix`. 
Is that correct? Can you also confirm what the links should be?

Reply:
This link is for `Ahk` & `Ahb` prefix
for the other one it looks like the page havn't been written, maybe @yangwao can help?